tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -89,6 +89,24 @@ test.describe( 'Price Benchmark Page', () => {
 			await expect( comparisonChartElement ).not.toBeVisible();
 		} );
 
+		test( 'Does not render the chart if Market Insights not enabled for the account.', async () => {
+			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions( [] );
+			await priceBenchmarkPage.fulfillPriceBenchmarkSummary( [], 403 );
+
+			const errorMessage = page.locator(
+				'.components-snackbar__content'
+			);
+			await expect( errorMessage ).toBeVisible();
```

Comment:
@mukeshpanchal27 If we are gracefully handling the error, then no error message should appear if the summary endpoint returns a 403 response code. No snackbar content should be visible. Can you double check please? Thanks!

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/2922/commits/e4aa0eaa8c72152ca7af6ddba1fff260ce7bcb63